products/antixlinux.md

```diff
@@ -0,0 +1,68 @@
+---
+permalink: /antix
+alternate_urls:
+-   /antixlinux
+-   /antix-linux
+title: antiX
+category: os
+releasePolicyLink: https://www.antixforum.com/forums/topic/when-is-end-of-support-for-stable-antix-versions-17-19/#post-26424
+iconSlug: mxlinux
+activeSupportColumn: true
+versionCommand: cat /etc/os-release
+releaseColumn: false
+releaseDateColumn: true
+realseLabel: "__RELEASE_CYCLE__ (__CODENAME__)"
```

Comment:
@alexkoster typo releaseLabel
Also if you put that long codenames it will look so messy in output

Reply:
you will understand what i mean in preview after you fix releeaseLabel typo above

Reply:
@usta Good suggestion and good catch.  I corrected the releaseLabel line and reduced the codenames to a single name in the latest update to the [antix branch](https://github.com/alexkoster/endoflife.date/tree/antix).  Thanks again for your time and please let me know if you'd like me to make any further changes.